all.js

```diff
@@ -519,9 +519,13 @@ function updateGlobalConfig(postfix, template) {
 /**
  * Writes app/config.h file (for C and Arduino)
  */
-function writeConfigH() {
+function writeConfigH(containsWifiConfig) {
   if (config.iot_device_connection_string) {
-    var headerContent = 'static const char* connectionString = ' + '"' + config.iot_device_connection_string + '"' + ';';
+    var headerContent = `static const char* connectionString = "${config.iot_device_connection_string}";`;
+    if (containsWifiConfig) {
+      headerContent =
+        `${headerContent}${os.EOL}static const char* ssid="${config.wifi_ssid}";${os.EOL}static const char* pass="${config.wifi_password}";`;
+    }
     fs.writeFileSync('./app/config.h', headerContent);
```

Comment:
For simplicity, how about

if(config.wifi__ssid && config.wifi_password)
{
....
}

I've reported one bug to Zim, that this header file contains credentials and shouldn't be stored in the repo folder. I think Zim will find a solution for the bug. :)